docs/standard/serialization/system-text-json-how-to.md

```diff
@@ -506,40 +560,107 @@ Here's an example object to serialize and JSON output:
+  "Summary": "\u0436\u0430\u0440\u043A\u043E"
+}
+```
+
+### Serialize language character sets
+
+To serialize the character set(s) of one or more languages, specify [Unicode range(s)](xref:System.Text.Unicode.UnicodeRanges) when creating an instance of <xref:System.Text.Encodings.Web.JavaScriptEncoder?displayProperty=fullName>, as shown in the following example:
+
+```csharp
+using System.Text.Encodings.Web;
+using System.Text.Json;
+using System.Text.Unicode;
+```
 
 ```csharp
 var options = new JsonSerializerOptions
 {
-    PropertyNameCaseInsensitive = true,
+    Encoder = JavaScriptEncoder.Create(UnicodeRanges.Cyrillic, UnicodeRanges.GreekExtended)
 };
-var weatherForecast = JsonSerializer.Deserialize<WeatherForecast>(weatherForecast, options);
+json = JsonSerializer.Serialize(weatherForecast, options);
 ```
 
-Here's example JSON with camel case property names. It can be deserialized into the following type that has Pascal case property names.
+This code serializes Cyrillic and Greek characters. Cyrillic characters are shown in the following example:
 
 ```json
 {
-  "date": "2019-08-01T00:00:00-07:00",
-  "temperatureC": 25,
-  "summary": "Hot",
+  "Date": "2019-08-01T00:00:00-07:00",
+  "TemperatureC": 25,
+  "Summary": "жарко",
 }
 ```
 
+To specify all languages, use <xref:System.Text.Unicode.UnicodeRanges.All?displayProperty=nameWithType>.
+
+### Serialize specific characters
+
+An alternative is to specify individual characters that you want to allow through without being escaped. The following example serializes only the first two characters of жарко:
+
+```csharp
+using System.Text.Encodings.Web;
+using System.Text.Json;
+using System.Text.Unicode;
+```
+
 ```csharp
-class WeatherForecast
+var encoderSettings = new TextEncoderSettings();
+encoderSettings.AllowCharacters('\u0436', '\u0430');
+options = new JsonSerializerOptions
 {
-    public DateTimeOffset Date { get; set; }
-    public int TemperatureC { get; set; }
-    public string Summary { get; set; }
+    Encoder = JavaScriptEncoder.Create(encoderSettings)
+};
+json = JsonSerializer.Serialize(weatherForecast, options);
+```
+
+Here's an example of JSON produced by the preceding code:
+
+```json
+{
+  "Date": "2019-08-01T00:00:00-07:00",
+  "TemperatureC": 25,
+  "Summary": "жа\u0440\u043A\u043E"
 }
 ```
 
-## Include properties of derived classes
+### Serialize all characters
+
+To minimize escaping you can use <xref:System.Text.Encodings.Web.JavaScriptEncoder.UnsafeRelaxedJsonEscaping?displayProperty=nameWithType>, as shown in the following example:
+
+```csharp
+using System.Text.Encodings.Web;
+using System.Text.Json;
+```
+
+```csharp
+options = new JsonSerializerOptions
+{
+    Encoder = JavaScriptEncoder.UnsafeRelaxedJsonEscaping
+};
+json = JsonSerializer.Serialize(weatherForecast, options);
+```
+
+> [!CAUTION]
+> Unlike the default encoder, the `UnsafeRelaxedJsonEscaping` encoder:
+>
+> * Doesn't escape HTML-sensitive characters such as `<`, `>`, and `&`. 
```

Comment:
I'd probably change the bullet points to read as follows:

 * Doesn't escape HTML-sensitive characters such as `<`, `>`, `&`, and `'`.
 * Doesn't offer any additional defense-in-depth protections against XSS or information disclosure attacks, such as those which might result from the client and server disagreeing on the _charset_.
 * Is more permissive than the default encoder on which characters may pass through unescaped.

Use the unsafe encoder only when it is known that the client will be interpreting the resulting payload as UTF-8 encoded JSON, such as if it is known that the server is sending the response header `Content-Type: application/json; charset=utf-8`. Never allow the raw `UnsafeRelaxedJsonEscaping` output to be emitted into an HTML page or a `<script>` element.